src/database.rs

```diff
@@ -51,6 +51,3 @@ impl Database {
         self.dbi
     }
 }
```

Comment:
This is to check during testing that `Send` and `Sync` are still auto-implemented by the compiler:

```suggestion
}

mod test {
    use super::*;

    // Just to check this compiles
    #[allow(unused)]
    fn database_is_send_sync(db: Database) {
        fn is_send_sync(_x: impl Send + Sync) {}
        is_send_sync(db)
    }
}
```

Reply:
Done